spec/import_spec.rb

```diff
@@ -346,21 +370,18 @@ def upload_file!(name, ext='csv')
           expect(Author.count).to eq(2)
         end
       end
-
     end
 
-
     context "with callback procs options" do
       let(:options) do
         {
-          before_import: ->(_) { true },
-          after_import: ->(_) { true },
-          before_batch_import: ->(_) { true },
-          after_batch_import: ->(_) { true }
+            before_import: ->(_) { true },
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.